### PR TITLE
feat: limit chat messages

### PR DESCRIPTION
**Pull request by eordano**

### CODE REVIEW

File: kernel/packages/ui/avatar/chatWindow.ts [hunk 1]
```diff
@@ -198,5 +199,9 @@ async function sendMsg(messageToSend: string) {
 
 function addMessage(messageEntry: MessageEntry): void {
   internalState.messages.push(messageEntry)
+  const length = internalState.messages.length
+  if (length > MAX_CHAT_MESSAGES) {
+    internalState.messages = internalState.messages.slice(length - MAX_CHAT_MESSAGES)
+  }
   createLogMessage(messagesLogStackContainer, messageEntry)
```
Comment (moliva) [inline kernel/packages/ui/avatar/chatWindow.ts]: We should be removing the message components from the UI as well, might need to keep track of these.

Comment (eordano) [reply kernel/packages/ui/avatar/chatWindow.ts]: oh right this is not React 😓  haha

File: kernel/packages/unity-interface/dcl.ts [hunk 2]
```diff
@@ -371,6 +371,40 @@ export const unityInterface = {
       gameInstance.SendMessage('SceneController', 'UpdateMinimapSceneInformation', JSON.stringify(chunk))
     }
   },
+  TriggerAirdropDisplay() {
```
Comment (BrianAmadori) [inline kernel/packages/unity-interface/dcl.ts]: I think this should be out of this PR.

**Review by BrianAmadori — DISMISSED** [review]

**Comment by BrianAmadori** [comment]
The logic works OK, however you can see how the messages processing starts moving messages up one by one in the world. That's can get annoying if a lot of messages are coming in.

**Pull request closed without merging**